test/features/user_records_an_application_test.rb

```diff
@@ -42,18 +46,20 @@ def test_viewing_the_details_of_an_application
   end
 
   def test_editing_the_details_of_an_application
-    app = Application.create!(:company => "Basecamp",
+    user = create(:person)
+    app = user.applications.create!(:company => "Basecamp",
                               :url => "http://basecamp.com",
                               :location => "Chicago, IL",
-                              :status => "open")
+                              :status => "applied")
 
+    page.set_rack_session(user_id: user.id)
     visit application_path(app)
     click_link_or_button("edit_application_#{app.id}")
 
     fill_in 'application_company', :with => "Trello"
     fill_in 'application_location', :with => "New York, NY"
     fill_in 'application_url', :with => "http://gettrello.com"
-    select "dead", :from => 'application_status'
+    select "closed", :from => 'application_status'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.